02-using-Shiny.Rmd

```diff
@@ -41,33 +43,37 @@ This checks the format of your workflow files to make sure you have a valid file
 <img src="assets/cromwell/validate.png" title="Validate a workflow" alt="Crowmell app" style="display: block; margin: auto;" />
 
 ### Submit a workflow
-This will let you upload the files that contain your workflow description (a WDL), and up to two different sets of input lists (in JSON form).  You can run a workflow with no input JSON, one input JSON, or two input JSONs (which will be concatenated or the second will overwrite the first if the same variable is declared in both).  You can upload a workflow options JSON, as well as providing text labels of your choosing to workflows if you'd like.  
+This will let you upload the files that contain your workflow description (a WDL), and up to two different sets of input lists (in JSON format).  You can run a workflow with no input JSON, one input JSON, or two input JSONs (which will be concatenated or the second will overwrite the first if the same variable is declared in both).  You can upload a workflow options JSON, as well as providing text labels of your choosing to workflows if you'd like.  
 <img src="assets/cromwell/cromwell-app.png" title="Submit a workflow" alt="Crowmell app" style="display: block; margin: auto;" />
 
+When you click that "SubmitWorkflow" button, you'll see confirmation in a new box that appears with the workflow submission ID and status. These IDs are long strings that look something like this: `4e7e244a-d6b1-41db-a324-45229ff34b00` and they're useful if for example you want to abort a workflow, or identify it in the "Track jobs" tab.
```

Comment:
Edit: "When you click that "Submit Workflow" button, you'll see confirmation in a new box that appears with the workflow submission ID and status. These IDs are long strings that look something like this: `4e7e244a-d6b1-41db-a324-45229ff34b00` and they're useful if for example you want to abort a workflow, or identify it in the "Track jobs" tab.  This workflow id string is unique to an individual workflow run, so if you run the same workflow a second time, you'll get a different string, meaning, this string can be used as an identifier for the data provenance of results files when reproducibility is critical.  "  Or something to that effect.

Reply:
yes, good.   I reworded that second part to make it more understandable to me, but let me know if it's no longer accurate: "This workflow id string is unique to an individual workflow run, so if you run the same workflow a second time, you'll get a different string. This means that this unique identifier string can be used to help understand the data source file(s) used to generate each set of results files, helping make your work reproducible."

Reply:
yesssssssss